The default video renderers (softx / soft) currently do not work 
on amd64.  So for now we add a little hack which will select the 
glx renderer by default.

http://bugs.gentoo.org/show_bug.cgi?id=86552

--- src/linux/vid_so.c
+++ src/linux/vid_so.c
@@ -435,6 +440,9 @@ VID_Init
 */
 void VID_Init (void)
 {
+#if defined __x86_64__
+	vid_ref = Cvar_Get ("vid_ref", "glx", CVAR_ARCHIVE);
+#endif
 	/* Create the video variables so we know how to start the graphics drivers */
 	// if DISPLAY is defined, try X
 	if (getenv("DISPLAY"))
